userspace/libsinsp/events/sinsp_events_set.h

```diff
@@ -155,11 +148,6 @@ class set
 		return m_types.data();
 	}
 
-	uint8_t* data() noexcept
```

Comment:
Avoid letting other write in our internal vector, since they won't update size counter.